_posts/2022-10-15-rtl.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: post
+title: Right to left scripting
+date: 2022-10-15 
+description: you can also write from right to left
+lang: fa
+tags: internationalization formatting math code
+category: internationalization
+---
+
+# عنوان
+این یک پست نمونه برای سنجش نوشتارهای راست‌به‌چپ است.
+
+
+## فرمول‌نویسی
+فرمول‌ها هم مانند کد به درستی به نمایش در‌خواهند آمد:
+
+
+$$
+\sum_{k=1}^\infty |\langle x, e_k \rangle|^2 \leq \|x\|^2
+$$
+
+## کد
+کدها هم‌چنان مانند قبل از چپ‌به‌راست خواهند بود.
+
+```python
+if lang=="rtl"
+    print("I am a right-to-left script!")
+```
+
+## بلاک
+بلاک‌ها همواره چپ‌چین خواهند ماند:
+
+    ---
+    layout: page
+    title: project
+    description: a project with a background image
+    img: /assets/img/12.jpg
+    ---
+    
+حتی اگر به زبان‌های راست‌به‌چپ نوشته شوند:
+    
+    ---
+    صفحه‌بندی: صفحه
+    عنوان: پروژه
+    توضیحات: یک پروژه با تصویر زمینه
+    ---
+    
+## نقل‌قول
+> نقل‌قول‌ها به این شکل به نمایش درخواهند آمد.
```

Comment:
I don't think blockquote renders correctly (the border is still on the left side)

Reply:
I agree. As a matter of fact, the content of this post addresses this issue (although in Persian -- if you use chrome or firefox you may want to translate the whole page to English. Translation is decent in my experience). That said, I was unsuccessful in fixing this issue. Maybe adding `rtl` to `div` resolves this.

Reply:
It is entirely fixed in the latest commit.